doc/Quick-Start-Guide.rst

```diff
@@ -74,7 +74,7 @@ You can also build RPMs by running the following command.
 
 - **$ make rpms**
 
-The following RPMs are generated in the **$HOME/rpmbuild/RPMS/x86_64** directory.
```

Comment:
No.

If you run "make rpms" with a normal user, the resulting rpms will be in your HOME dir.
If you run "sudo make rpms", the generated rpms will be in /root/, that is the HOME of root user.